src/LSST/control/Context.cpp

```diff
@@ -38,12 +39,15 @@ Context::Ptr Context::_thisPtr;
 std::mutex Context::_thisMtx;
 
 void Context::setup() {
-    lock_guard<mutex> lock(_thisMtx);
-    if (_thisPtr) {
-        LERROR("Config already setup");
-        return;
+    {
+        lock_guard<mutex> lock(_thisMtx);
+        if (_thisPtr) {
+            LERROR("Config already setup");

```

Comment:
Is `Context` the same as `config`, or do they both refer to the same thing? Asking this question since I noticed that `_thisPtr` is a pointer to an object of the class `Context` while the error message above implies something else.

Reply:
Comment fixed.